MAINTAINERS.md

```diff
@@ -7,13 +7,23 @@
   - [Triage Open Issues](#triage-open-issues)
   - [Be Responsive](#be-responsive)
   - [Maintain Overall Health of the Repo](#maintain-overall-health-of-the-repo)
-  - [Add Continious Integration Checks](#add-continious-integration-checks)
+  - [Add Continuous Integration Checks](#add-continuous-integration-checks)
     - [Developer Certificate of Origin Workflow](#developer-certificate-of-origin-workflow)
   - [Use Semver](#use-semver)
   - [Release Frequently](#release-frequently)
   - [Promote Other Maintainers](#promote-other-maintainers)
   - [Describe the Repo](#describe-the-repo)
-  
+- [Becoming a Maintainer](#becoming-a-maintainer)
+  - [Nomination](#nomination)
+  - [Interest](#interest)
+  - [Public Confirmation](#public-confirmation)
+  - [Maintainer Decision](#maintainer-decision)
+  - [Adding the New Maintainer](#adding-the-new-maintainer)
```

Comment:
Nit : It should be `Adding a New Maintainer` right?

Reply:
Corrected.